Cookbook/Proposals/ComposableReducers.md

```diff
@@ -0,0 +1,181 @@
+# Composable Reducers using Lenses and Prisms
+
+* Author: Yasuhiro Inami
+* Review Manager: David Rodrigues
+
+## Introduction
+
+This proposal adds functional `Lens` and `Prism` ideas to break down `reducer` / `state` / `event` (`action`) in [ReactiveFeedback](https://github.com/Babylonpartners/ReactiveFeedback) so that we can create a large application from small component compositions.
+
```

Comment:
Please explain what `Lens` and `Prisms` are.

Reply:
5390a38 Linked to proposed solution which describes the functionality.